src/Core/Attribute/Configure.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace Cake\Core\Attribute;
+
+use Attribute;
+use Cake\Core\Configure as CakeConfigure;
+use League\Container\Attribute\AttributeInterface;
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
+
+#[Attribute(Attribute::TARGET_PARAMETER)]
+class Configure implements AttributeInterface, ContainerAwareInterface
```

Comment:
It would be good to have a doc-block with a short description of how to use this, perhaps an example?